TSC/core-and-hosted-projects.md

```diff
@@ -20,34 +20,41 @@ Core projects must further the mission of the Bytecode Alliance and be seen as v
-  * Fill in all of the `TODO`s.
-  * Do not make unsubstantiated claims; provide evidence. For example, if you claim that your project has a CI job running tests under [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), provide links to its CI configuration, test infrastructure, and bugs caught.
+* [ ] Create a Pull Request against [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance) with the following changes:
+  * a copy of [/templates/projects/proposal-core-project.md](../templates/projects/proposal-core-project.md), with all of the `TODO`s filled in, as a new file under `/projects/core/$PROJECT_NAME/proposal-core.md`
+    * Do not make unsubstantiated claims; provide evidence. For example, if you claim that your project has a CI job running tests under [AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), provide links to its CI configuration, test infrastructure, and bugs caught.
+  * moving any existing files from `/projects/hosted/$PROJECT_NAME` to `/projects/core/$PROJECT_NAME`
+    * If a `proposal.md` exists for the project, move it to `/projects/core/$PROJECT_NAME/proposal-hosted.md`
+  * An update to `/projects/README.md` to move the project to the list of core projects.
 
 The Bytecode Alliance Technical Steering Committee (TSC) will then perform the following steps:
 
 * [ ] Provide timely acknowledgment of the application.
 * [ ] Perform an in-depth review of the project:
   * The TSC will consider the strategic value of the project to the Bytecode Alliance, how it furthers the Bytecode Alliance's mission, as well as its value and influence in the broader community.
   * The TSC will evaluate whether it meets the technical requirements for core projects.
-  * Finally, the TSC will decide based on consensus whether to promote the project.
-* [ ] Reply in the issue with the results of the review, providing any feedback they might have, and whether the project has been promoted.
-* [ ] If it has been promoted, then the project maintainers will [choose a delegate to represent the project on the TSC](https://github.com/bytecodealliance/governance/blob/d0310b2f4fab7d5bd076627b196dfc0ffe040ec1/TSC/charter.md#composition).
+  * Finally, the TSC will make a recommendation based on consensus whether to promote the project and work with the Board of Directors to decide on the promotion.
+* [ ] Reply in the Pull Request with the results of the review, providing any feedback they might have.
+* [ ] [The Board will make a decision](charter.md?rgh-link-date=2024-12-30T14%3A30%3A48Z#core-and-hosted-projects) on the promotion and reply in the Pull Request.
+* [ ] If it has been promoted,
+  * [ ] the project maintainers will [choose a delegate to represent the project on the TSC](charter.md?rgh-link-date=2024-12-30T14%3A30%3A48Z#composition).
```

Comment:
Are we sure we want to keep these `rgh-link-date` modifiers in? What are they for? Seems like we want this link to point to the latest version, not some older version.

Reply:
`rgh-link-date` doesn't show the old content. Instead, it makes GH add a header `You can also view this object as it appeared at the time of the comment ([some time period] ago)`, with the part starting at `view this object` linked to the old version.